modular_azurepeak/code/modules/jobs/job_types/roguetown/vagabond/vagabond.dm

```diff
@@ -19,6 +19,6 @@
 		H.advsetup = 1
 		H.invisibility = INVISIBILITY_MAXIMUM
 		H.become_blind("advsetup")
-
+		H.remove_teeth(rand(0, 10))
```

Comment:
punishing vagabond players with the random possibility of being nearly impossible to understand kind of sucks given how expensive replacements can be